### PR TITLE
chore: change Support Question and Bug Report templates

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -33,9 +33,11 @@ about: Report a bug to help us improve
 
 ### Steps to Reproduce
 <!--
-    Provide steps to reproduce the bug. If possible, create a demo illustrating the problem.
+    Provide steps to reproduce the bug.
 -->
 
+I have created my example based on [CodeSandbox examples](https://github.com/DevExpress/devextreme-reactive/issues/2358).
```
Comment (RomanTsukanov) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -33,9 +33,11 @@ about: Report a bug to help us improve
 
 ### Steps to Reproduce
 <!--
-    Provide steps to reproduce the bug. If possible, create a demo illustrating the problem.
+    Provide steps to reproduce the bug.
```
Comment (RomanTsukanov) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
    Create an example based on one of the [CodeSandbox examples](https://github.com/DevExpress/devextreme-reactive/issues/2358) and provide steps to reproduce the bug.
```

File: .github/ISSUE_TEMPLATE/support_question.md [hunk 3]
```diff
@@ -26,6 +26,8 @@ about: Ask a question about our product
     Ask your question. Provide as many details as possible.
 -->
 
+I have created my example based on [CodeSandbox examples](https://github.com/DevExpress/devextreme-reactive/issues/2358).
```
Comment (RomanTsukanov) [inline .github/ISSUE_TEMPLATE/support_question.md]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/support_question.md [hunk 4]
```diff
@@ -26,6 +26,8 @@ about: Ask a question about our product
     Ask your question. Provide as many details as possible.
```
Comment (RomanTsukanov) [inline .github/ISSUE_TEMPLATE/support_question.md]:
```suggestion
    Ask your question. Provide as many details as you can. If possible, create an example based on one of the [CodeSandbox examples](https://github.com/DevExpress/devextreme-reactive/issues/2358) to illustrate your question.
```

**Comment by AryamnovEugeniy** [comment]
FPR

**Pull request merged**